### ISSUE TITLE
Change Transaction name to request URL instead of HandlerName in nrgin

### ISSUE THREAD

**Issue by swaroopdeval** [newrelic/go-agent#126]
In nrgin Middleware, transaction name is HandlerName, can we change it to request URL? It will give more visibility.

This is the router code:
```
config := newrelic.NewConfig(
    viper.GetString("newrelic.name"),
    viper.GetString("newrelic.license"),
)
app, _ = newrelic.NewApplication(config)
r.Use(nrgin.Middleware(app))
r.GET("/ping", pingHandler())
```

For v1 nrgin set transaction name like this:`name := c.HandlerName()` and its very similar in V3. This requests in transition name like: `/github.com/username/repo/router.pingHandler.func1`.  Can we make transaction name to '/ping' as it is more clear.

**Comment by willnewrelic** [comment]
Hi @swaroopdeval 

Thanks for writing in.  We will discuss this.

**Comment by purple4reina** [comment]
@swaroopdeval 

We just released v3.4.0 of the Go Agent which addresses this issue. Thanks a ton for the suggestion!

~ Rey
